pkg/channel/consolidated/dispatcher/dispatcher.go

```diff
@@ -90,13 +89,15 @@ func (sub Subscription) String() string {
 
 func NewDispatcher(ctx context.Context, args *KafkaDispatcherArgs) (*KafkaDispatcher, error) {
 	conf := sarama.NewConfig()
-	conf.Version = sarama.V2_0_0_0
+	// don't get the clientID cleared by when it is built later
 	conf.ClientID = args.ClientID
-	conf.Consumer.Return.Errors = true    // Returns the errors in ConsumerGroup#Errors() https://godoc.org/github.com/Shopify/sarama#ConsumerGroup
-	conf.Producer.Return.Successes = true // Must be enabled for sync producer
+	// explicit version
+	conf.Version = sarama.V2_0_0_0
+
+	client.InitSaramaConfig(conf)
 
 	// append the auth info:
-	err := source.UpdateSaramaConfigWithKafkaAuthConfig(conf, args.KafkaAuthConfig)
+	conf, err := client.BuildSaramaConfig(conf, "", args.KafkaAuthConfig)
```

Comment:
Once we decide to use the Sarama settings from `config-kafka`, we can simply pass it here (2nd arg).